resources/js/views/settings/Application.vue

```diff
@@ -0,0 +1,471 @@
+                      v-model="settings.ownRoomsPaginationPageSize"
+                      type="number"
+                      :disabled="isBusy || viewOnly"
+                      :state='fieldState("own_rooms_pagination_page_size")'
+        >
+        </b-form-input>
+
+        <template slot='invalid-feedback'>
+          <div v-html="fieldError('own_rooms_pagination_page_size')"></div>
+        </template>
+      </b-form-group>
+
+    </b-container>
+</template>
+
+<script>
+import Base from '../../api/base';
+import FieldErrors from '../../mixins/FieldErrors';
+import env from '../../env';
+import PermissionService from '../../services/PermissionService';
+
+export default {
+  mixins: [FieldErrors],
+
+  data () {
+    return {
+      roomLimitMode: 'custom',
+      uploadLogoFile: null,
+      uploadLogoFileSrc: null,
+      uploadFaviconFile: null,
+      uploadFaviconFileSrc: null,
+      isBusy: false,
+      settings: {
+        name: null,
+        logo: null,
+        favicon: null,
+        roomLimit: null,
+        paginationPageSize: null,
+        ownRoomsPaginationPageSize: null
+      },
+      errors: {}
+    };
+  },
+  methods: {
+    /**
+     * Handle get settings data
+     */
+    getSettings () {
+      this.isBusy = true;
+      Base.call('settings')
+        .then(response => {
+          this.settings.name = response.data.data.name;
+          this.settings.logo = response.data.data.logo;
+          this.settings.favicon = response.data.data.favicon;
+          this.settings.roomLimit = response.data.data.room_limit;
+          this.settings.ownRoomsPaginationPageSize = response.data.data.own_rooms_pagination_page_size;
+          this.settings.paginationPageSize = response.data.data.pagination_page_size;
+          this.roomLimitMode = (this.settings.roomLimit === -1 ? 'unlimited' : 'custom');
+        })
+        .catch((error) => {
+          Base.error(error, this.$root, error.message);
+        })
+        .finally(() => {
+          this.isBusy = false;
+        });
+    },
+
+    /**
+     * Handle update settings data
+     *
+     * @param settings Settings object to fill the payload
+     */
+    updateSettings (settings) {
+      this.isBusy = true;
+
+      // Build form data
+      const formData = new FormData();
+      if (this.uploadLogoFile) {
+        formData.append('logo_file', this.uploadLogoFile);
+      } else {
+        formData.append('logo', settings.logo);
+      }
+      if (this.uploadFaviconFile) {
+        formData.append('favicon_file', this.uploadFaviconFile);
+      } else {
+        formData.append('favicon', settings.favicon);
+      }
+      formData.append('name', settings.name);
+      formData.append('room_limit', settings.roomLimit);
+      formData.append('pagination_page_size', settings.paginationPageSize);
+      formData.append('own_rooms_pagination_page_size', settings.ownRoomsPaginationPageSize);
+      formData.append('_method', 'PUT');
```

Comment:
Why here is the method put and below in the call post?

Reply:
https://github.com/laravel/framework/issues/13457